### PR TITLE
feat(dsg): generated app non privileged user

### PR DESCRIPTION
**Pull request by levivannoort**
Close: #5060

## PR Details

Change the Dockerfiles in the static part of the admin & server of the data service generator to reflect the need of the non-privileged user on running the image. On the admin side this also required adding a config directory which includes an nginx.conf without a user specification on top.

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by levivannoort** [comment]
@overbit could you try to copy these files over to a generated application and try and run the components based on the new Dockerfiles.

File: packages/data-service-generator/src/server/static/Dockerfile [hunk 1]
```diff
@@ -26,6 +51,18 @@ COPY --from=base /app/scripts ./scripts
...
+
+# get rid of the development dependencies
+RUN npm install --production
+
+# set user to the created non-privileged user
+USER ${user}
+
+# expose a specific port on the docker container
+ENV PORT=3000
+EXPOSE ${PORT}
```
Comment (overbit) [inline packages/data-service-generator/src/server/static/Dockerfile:66]: nice touch! 👍

**Review by overbit — APPROVED** [review]

**Comment by levivannoort** [comment]
This is a derivative of the work in: https://github.com/amplication/amplication/pull/5065

**Review by overbit — APPROVED** [review]
💯

**Pull request merged**